tests/workloads/nixl_ep/test_job_status_retrieval_strategy.py

```
@@ -69,6 +69,76 @@ def test_successful_job(self, nixl_ep_tr: TestRun) -> None:
+            (nixl_ep_tr.output_path / f"nixl-ep-node-{node_idx}.log").write_text(
+                SUCCESSFUL_BANDWIDTH_LINE, encoding="utf-8"
+            )
+        (nixl_ep_tr.output_path / "stderr.txt").write_text(
+            "\n".join(
+                [
+                    "srun: error: node001: task 0: Terminated",
+                    "Traceback (most recent call last):",
+                    "srun: Terminating StepId=123.4",
+                    "srun: Force Terminated StepId=123.4",
+                ]
+            )
```

Ruff (0.15.15)

[warning] Consider f-string instead of string join

Replace with f-string

(FLY002)